docs/snippets_table.py

```diff
@@ -692,6 +693,33 @@ def test_bigtable_add_row_add_row_range_add_row_range_from_keys():
     found_row_keys = [row.row_key for row in read_rows]
     assert found_row_keys == expected_row_keys
 
+    # [START bigtable_row_range_by_prefix_from_keys]
+    from google.cloud.bigtable import Client
+    from google.cloud.bigtable.row_set import RowSet
+
+    client = Client(admin=True)
+    instance = client.instance(INSTANCE_ID)
+    table = instance.table(TABLE_ID)
+
+    row_set = RowSet()
+    row_set.add_row_range_by_prefix_from_keys("row")
+    # [END bigtable_row_range_by_prefix_from_keys]
+
+    read_rows = table.read_rows(row_set=row_set)
+    expected_row_keys = [
+        b"row_key_1",
+        b"row_key_2",
+        b"row_key_3",
+        b"row_key_4",
+        b"row_key_5",
+        b"row_key_6",
+        b"row_key_7",
+        b"row_key_8",
+        b"row_key_9",
+    ]
+    found_row_keys = [row.row_key for row in read_rows]
```

Comment:
IMO, a more elegant and concise way would be
```
expected_row_keys = [b"row_key_%x" % i for i in range(1, 10)]
found_row_keys = [row.row_key for row in table.read_rows(row_set=row_set)]
```

Reply:
Yes , correct but they have declare static before in snippets and system file so ta make consistent i made this static.